sdk-tests/src/test/java/io/dapr/it/testcontainers/DaprContainerTest.java

```diff
@@ -97,19 +93,23 @@ private static void configStub() {
 
   @Test
   public void testDaprContainerDefaults() {
+    assertEquals(2,
+        DAPR_CONTAINER.getComponents().size(),
+        "The pubsub and kvstore component should be configured by default"
+    );
     assertEquals(
-        "The pubsub and kvstore component should be configured by default",
-        2,
-        daprContainer.getComponents().size());
-    assertEquals(
-        "A subscription should be configured by default if none is provided",
         1,
-        daprContainer.getSubscriptions().size());
+        DAPR_CONTAINER.getSubscriptions().size(),
+        "A subscription should be configured by default if none is provided"
+    );
   }
 
   @Test
   public void testStateStore() throws Exception {
-    try (DaprClient client = (new DaprClientBuilder()).build()) {
+    DaprClientBuilder builder = new DaprClientBuilder()
+        .withPropertyOverride(Properties.HTTP_PORT, String.valueOf(DAPR_CONTAINER.getHttpPort()))
+        .withPropertyOverride(Properties.GRPC_PORT, String.valueOf(DAPR_CONTAINER.getGrpcPort()));
+    try (DaprClient client = (builder).build()) {
       client.waitForSidecar(5000).block();
```

Comment:
@artur-ciocanu can we just use 1s all over the place? If we are waiting for more than 1 second then something is wrong somewhere else.